docs/product-updates/2023-09-28-manual-release.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Introducing Manual Release Mode
+tags: [New]
+image: /files/changelog/manual-release.png
+---
+
+![change-comparison.png](/files/changelog/manual-release.png)
+
+Gain greater control over your API Documentation deployments by using the Manual Release Mode for your API documentation.
+
+**Grouping of Changes:** Multiple changes are sometimes part of a single feature or improvement. Manual releases allow these changes to be grouped together under a single changelog entry, rather than being scattered across multiple automatically generated entries.
+
+**Timing:** Not all changes may be immediately relevant to API consumers. The manual release option allows the team to decide when a particular change should be communicated, be it immediately or bundled with other changes at a later date.
```

Comment:
Should we add an example here, to show the power of the feature? Something like "You continue merging your PRs and following your API changes on a daily basis, but you control when you communicate these changes to you API users". WDYT?

Reply:
Updated ✅  wdyt?